Documentation/csiaddons.md

```diff
@@ -0,0 +1,93 @@
+# CSI-Addons for Kubernetes
```

Comment:
Instead of a new topic, with the simplifications suggested by @nixpanic it seems sufficient to add a single new section to the ceph-csi-drivers.md.